db_schema/migrations/12_anchor_impressions.sql

```diff
@@ -0,0 +1,38 @@
+-- Create table for storing aggregate impression metrics
+CREATE TABLE IF NOT EXISTS anchorImpressions (
+    account_id INTEGER NOT NULL,
+    date_start TIMESTAMP NOT NULL,
+    date_end TIMESTAMP NOT NULL,
+    total_impressions INTEGER NOT NULL,
+    total_considerations INTEGER NOT NULL,
+    total_streams INTEGER NOT NULL,
+    considerations_conversion_rate DECIMAL(10,5),
+    streams_conversion_rate DECIMAL(10,5),
+    PRIMARY KEY (account_id, date_start, date_end)
+);
+
+-- Create table for daily impression data
+CREATE TABLE IF NOT EXISTS anchorImpressionsDaily (
+    account_id INTEGER NOT NULL,
+    date TIMESTAMP NOT NULL,
+    impressions INTEGER NOT NULL,
+    PRIMARY KEY (account_id, date)
+);
+
+-- Create table for impression sources breakdown
+CREATE TABLE IF NOT EXISTS anchorImpressionsSources (
+    account_id INTEGER NOT NULL,
+    date_start TIMESTAMP NOT NULL,
+    date_end TIMESTAMP NOT NULL,
+    source_id VARCHAR(32) NOT NULL,
+    source_name VARCHAR(64) NOT NULL,
```

Comment:
I think we could skip the source name as it is just for the frontend. the id would be sufficient. the best would be to have an enum here but that we might lose assitional dimensions that are added in the feature. or we have proper error handling ;)

Reply:
Hm, I would keep it. The column is useful for reports and consistent with the API data structure. 🤔